homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css

```diff
@@ -0,0 +1,17 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+h1 {
+  font-family: Roboto;
+  font-size: 48px;
+  font-weight: 700;
+}
+
+.main__content {
+  font-family: Roboto;
+  font-size: 22px;
+  font-weight: 300;
+}
+
+p a {
+  color: #EF4934;
+}
```

Comment:
Better use classes instead tags for styling.
And read pls about methodology BEM and use this pls.

Reply:
Okay. Thank you I did it)

Reply:
You still use tag a for styling) pls add class to this tag. 
Something like this:
.link {
  color: #EF4934;
}

Reply:
Okay I did it, thank you